frontend/views/containers/sidebar/GroupsMinIncome.vue

```diff
@@ -7,7 +7,7 @@ div
 
   i18n.link(
     tag='button'
-    aria-label="Change minIncome"
+    aria-label='Change minIncome'
```

Comment:
While we're modifying this, we might as well and boy-scout this to make sure that all user-facing strings are translated, so this should read:

```
:aria-label='L("Change Mincome")'
```

Reply:
done